packages/create-instance/create-component-stubs.js

```diff
@@ -110,36 +127,53 @@ export function createStubFromComponent(
   // ignoreElements does not exist in Vue 2.0.x
   if (Vue.config.ignoredElements) {
     Vue.config.ignoredElements.push(tagName)
+    if (Vue.config.ignoredElements.indexOf('template-stub') === -1) {
+      Vue.config.ignoredElements.push('template-stub')
+    }
   }
 
   return {
     ...getCoreProperties(componentOptions),
     $_vueTestUtils_original: originalComponent,
     $_doNotStubChildren: true,
     render(h, context) {
+      const attrs = getAttributes(componentOptions.functional, context, this)
+      const slots = context ? context.slots() : this._renderProxy.$slots
+
+      // ensure consistent ordering of slots (default first, then alphabetical)
+      const sortedSlotEntries = Object.entries(slots)
+      sortedSlotEntries.sort(([slotNameA], [slotNameB]) =>
+        slotNameA === 'default'
+          ? -1
+          : slotNameB === 'default'
+          ? 1
+          : slotNameA.localeCompare(slotNameB)
+      )
+
+      const sortedSlots = sortedSlotEntries.map(([slotName, slotChildren]) =>
+        slotName === 'default'
+          ? slotChildren
+          : h('template-stub', { attrs: { slot: slotName } }, slotChildren)
+      )
+
+      // let children
+      // if (context) {
+      //   children = sortedSlots
+      // } else if (this.$options._renderChildren) {
+      //   children = this.$options._renderChildren
+      // } else {
+      //   children = getScopedSlotRenderFunctions(this).map(x =>
+      //     this.$options.parent._vnode.data.scopedSlots[x]()
+      //   )
+      // }
+
```

Comment:
This was initially how I expanded out and merged dev and this PR, but was failing so I commented it out and reverted to mainly this PR's version of rendering slots